SparkPost/matchbox
# FE-1065 Fix Drawer.Footer position to bottom

### What Changed
- Reverts a portion of this commit to add footer height calculation back in https://github.com/SparkPost/matchbox/pull/384/commits/0954f57d15b2e089b886eca7226d49d983e97721

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Visit Drawer stories, verify footer is always on the bottom of the drawer

### PR Checklist
